### PR TITLE
Release Notes 2.34.0

### PR DESCRIPTION
**Pull request by rasika-chivate**
1. Create Release Notes for 2.34.0
2. Update TOC

### CODE REVIEW

File: docs/release-notes/2.34.0.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    
+
+- [PMM-11031](https://jira.percona.com/browse/PMM-11031) - By using the Expose field, DBaaS users can easily connect a database cluster in a secure environment. You can modify the exposure level using the Source Range field.
+- [PMM-11086](https://jira.percona.com/browse/PMM-11086) - Improved design for the **On Demand** and **Schedule Backup** cards on the **Create Backup** page. In addition, the **Created** column on the **Restores** tab has been replaced with **Started at** and **Ended** columns, to provide more insight into the restore process. 
+
+
+## Improvements
+
+- [PMM-11024](https://jira.percona.com/browse/PMM-11024) - [DBaaS]: Move from dbaas-controller to dbaas-operator
```
Comment (rasika-chivate) [inline docs/release-notes/2.34.0.md:42]: @gen1us2k What is the user impact for this ticket?

File: docs/release-notes/2.34.0.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+- [PMM-11086](https://jira.percona.com/browse/PMM-11086) - Improved design for the **On Demand** and **Schedule Backup** cards on the **Create Backup** page. In addition, the **Created** column on the **Restores** tab has been replaced with **Started at** and **Ended** columns, to provide more insight into the restore process. 
+
+
+## Improvements
+
+- [PMM-11024](https://jira.percona.com/browse/PMM-11024) - [DBaaS]: Move from dbaas-controller to dbaas-operator
+- [PMM-10873](https://jira.percona.com/browse/PMM-10873) - [DBaaS]: Create DB Cluster : Move from Popup to the separate Page
+- [PMM-11134](https://jira.percona.com/browse/PMM-11134) - [DBaaS]: Edit DB Cluster : Move from Popup to the separate Page
+- [PMM-11183](https://jira.percona.com/browse/PMM-11183) - [DBaaS]: Default values in PMM DBaaS in Operators
```
Comment (rasika-chivate) [inline docs/release-notes/2.34.0.md:48]: @gen1us2k Could you please provide a short description here?

File: docs/release-notes/2.34.0.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Bugs Fixed
+
+- [PMM-7350](https://jira.percona.com/browse/PMM-7350) - [Dbaas]: Fixed an issue where DB Cluster did not restart all PXC servers and proxy SQL servers on restarting.
+- [PMM-7958](https://jira.percona.com/browse/PMM-7958) - During Postgres monitoring, databases were not getting deleted. This issue has been resolved now.
+- [PMM-9505](https://jira.percona.com/browse/PMM-9505) - [DBaaS]: The options to update, restart, and view logs were enabled for paused clusters. This issue has now been resolved by disabling these options.
+- [PMM-9977](https://jira.percona.com/browse/PMM-9505) - [Backup]: When deleting a backup artifact that was expanded from **Backup > All Backups**, PMM would automatically expand the next row in the table. This issue is now fixed and PMM refreshes the list of backups without automatically expanding any rows.
+- [PMM-10853](https://jira.percona.com/browse/PMM-10853) - [IA] : Duplicate Buttons for Saving rule based on Percona Templates
```
Comment (rnovikovP) [inline docs/release-notes/2.34.0.md:62]:
```suggestion
- [PMM-10853](https://jira.percona.com/browse/PMM-10853) - [Alerting] : Duplicate Buttons for Saving rule based on Percona Templates
```

File: docs/release-notes/2.34.0.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- List the components which are upgraded as part of the Release.
+
+## New Features
+  
+- [PMM-11031](https://jira.percona.com/browse/PMM-11031) - To enhance the DBaas experience, the UI has been extended to include the following:
+    - Network and Security - DBaaS users can set a source range from which they can connect to their DB clusters with the **Expose** field. You can modify the exposure level using the **Source Range** field. Additionally, the **Source Range** field allows you to choose the network from which you can connect.
+    - Configuring the database engine - Copy-paste the config string into the **Configuration** text box to configure the database engine.
+    - Configuring storage classes - You can choose from the available classes.
+
+- [PMM-11086](https://jira.percona.com/browse/PMM-11086) - Improved design for the **On Demand** and **Schedule Backup** cards on the **Create Backup** page. In addition, the **Created** column on the **Restores** tab has been replaced with **Started at** and **Ended** columns to provide more insight into the restore process.
```
Comment (rnovikovP) [inline docs/release-notes/2.34.0.md:34]: This probably needs to be "Improvements" as not something completely new, but an improvement to the existing presentation.  

@artemgavrilov ?

File: docs/release-notes/2.34.0.md (hunk 1 above)
Comment (gen1us2k) [reply docs/release-notes/2.34.0.md:39]: I'll add notes

File: docs/release-notes/2.34.0.md (hunk 4 above)
Comment (artemgavrilov) [reply docs/release-notes/2.34.0.md:34]: Yes, for sure, it's just an improvement

Comment (rasika-chivate) [reply docs/release-notes/2.34.0.md:45]: ok

**Review by dygksy — APPROVED** [review]

File: docs/release-notes/2.34.0.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+- [PMM-9079](https://jira.percona.com/browse/PMM-9079)   - [Backups]: Added **Target Service** column under **Backup > Restores** to show the service to which a backup was restored.
+- [PMM-10773](https://jira.percona.com/browse/PMM-10773) - [Alerting]: Added option to unsilence alerts from the **Actions** menu under **Alerting > Fired Alerts**.
+- [PMM-11230](https://jira.percona.com/browse/PMM-11230) - [Alerting]: The **Fired alerts** page under **Alerting** now auto-refreshes more frequently to ensure that any new alerts are displayed in real-time.
+- [PMM-11255](https://jira.percona.com/browse/PMM-11255) - [Alerting]: PMM no longer transfers the name of the current alert rule when switching from **Percona templated alert** to **Grafana managed alert**.
+- [PMM-11255](https://jira.percona.com/browse/PMM-11255) - [FE]: Don't copy alert rule name when switching from Percona editor to Grafana editor
+- [PMM-11090](https://jira.percona.com/browse/PMM-11090) - The Logs feature has been moved from the dbaas-controller to PMM, providing users with the ability to see the events for each pod and the logs for every container properly populated within PMM.
```
Comment (BupycHuk) [inline docs/release-notes/2.34.0.md:62]: seems like it needs [DBaaS] prefix

File: docs/release-notes/2.34.0.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+- [PMM-9079](https://jira.percona.com/browse/PMM-9079)   - [Backups]: Added **Target Service** column under **Backup > Restores** to show the service to which a backup was restored.
+- [PMM-10773](https://jira.percona.com/browse/PMM-10773) - [Alerting]: Added option to unsilence alerts from the **Actions** menu under **Alerting > Fired Alerts**.
+- [PMM-11230](https://jira.percona.com/browse/PMM-11230) - [Alerting]: The **Fired alerts** page under **Alerting** now auto-refreshes more frequently to ensure that any new alerts are displayed in real-time.
+- [PMM-11255](https://jira.percona.com/browse/PMM-11255) - [Alerting]: PMM no longer transfers the name of the current alert rule when switching from **Percona templated alert** to **Grafana managed alert**.
+- [PMM-11255](https://jira.percona.com/browse/PMM-11255) - [FE]: Don't copy alert rule name when switching from Percona editor to Grafana editor
```
Comment (BupycHuk) [inline docs/release-notes/2.34.0.md:61]:
```suggestion
- [PMM-11255](https://jira.percona.com/browse/PMM-11255) - [Alerting]: Don't copy alert rule name when switching from Percona editor to Grafana editor
```

File: docs/release-notes/2.34.0.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+- [PMM-11090](https://jira.percona.com/browse/PMM-11090) - The Logs feature has been moved from the dbaas-controller to PMM, providing users with the ability to see the events for each pod and the logs for every container properly populated within PMM.
+- [PMM-11179](https://jira.percona.com/browse/PMM-11179) - Starting with PMM 2.34.0, we support push metrics mode while updating labels.
+- [PMM-11189](https://jira.percona.com/browse/PMM-11189) - Starting With PMM 2.34.0, we have implemented vmproxy integration and frontend API changes.
+- [PMM-10705](https://jira.percona.com/browse/PMM-10705) - As of PMM 2.34.0, Operator Lifecycle Management (OLM) is used for handling the installation and updating of operators for PMM in an effective, automated, and scalable manner.
```
Comment (BupycHuk) [inline docs/release-notes/2.34.0.md:65]:
```suggestion
- [PMM-10705](https://jira.percona.com/browse/PMM-10705) - [DBaaS]: As of PMM 2.34.0, Operator Lifecycle Management (OLM) is used for handling the installation and updating of operators for PMM in an effective, automated, and scalable manner.
```

File: docs/release-notes/2.34.0.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+## Bugs Fixed
+
+- [PMM-7350](https://jira.percona.com/browse/PMM-7350) - [Dbaas]: Fixed an issue where DB Cluster did not restart all PXC servers and proxy SQL servers on restarting.
```
Comment (BupycHuk) [inline docs/release-notes/2.34.0.md:69]:
```suggestion
- [PMM-7350](https://jira.percona.com/browse/PMM-7350) - [DBaaS]: Fixed an issue where DB Cluster did not restart all PXC servers and proxy SQL servers on restarting.
```

**Review by BupycHuk — APPROVED** [review]

File: docs/release-notes/2.34.0.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+Starting with PMM 2.34.0, PMM installs and manages operators using Operator Lifecycle Manager (OLM) and database clusters using `dbaas-operator`. Nonetheless, these changes are backward incompatible, and you must perform these additional steps:
+
+1. Unregister the Kubernetes cluster from PMM and register it back. PMM will install OLM and `dbaas-catalog` to manage the lifecycle of PXC, PSMDB, and DBaaS operators.
+2. Create a database cluster for each database cluster you have created in PMM. You can use the [script]() to do it automatically.
+
+For more information, see the [architecture]() of DBaaS.
+
+### New components
+
+VMProxy version 2.34.0 has been introduced as part of PMM 2.34.0.
```
Comment (denisok) [inline docs/release-notes/2.34.0.md:36]: I think we should rephrase this. Maybe that we just have a new component VMProxy. But also would be cool to see a small description why it is needed. @michal-kralik @dygksy

File: docs/release-notes/2.34.0.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+- Added information about the type of storage for the backup artifacts on the  **All backups** page. This helps you get a more comprehensive overview of your current backups.
+- Design and usability improvements on the **Create Backup** and **All Backups** page to increase consistency with other PMM UI controls.
+
+
+### DBaaS migration
+
+Starting with PMM 2.34.0, PMM installs and manages operators using Operator Lifecycle Manager (OLM) and database clusters using `dbaas-operator`. Nonetheless, these changes are backward incompatible, and you must perform these additional steps:
+
+1. Unregister the Kubernetes cluster from PMM and register it back. PMM will install OLM and `dbaas-catalog` to manage the lifecycle of PXC, PSMDB, and DBaaS operators.
+2. Create a database cluster for each database cluster you have created in PMM. You can use the [script]() to do it automatically.
```
Comment (denisok) [inline docs/release-notes/2.34.0.md:30]: @gen1us2k there is a link for the script is missing?

File: docs/release-notes/2.34.0.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+- [PMM-11255](https://jira.percona.com/browse/PMM-11255) - [FE]: Don't copy alert rule name when switching from Percona editor to Grafana editor
+- [PMM-11090](https://jira.percona.com/browse/PMM-11090) - The Logs feature has been moved from the `dbaas-controller` to PMM, providing users with the ability to see the events for each pod and the logs for every container properly populated within PMM.
+- [PMM-11179](https://jira.percona.com/browse/PMM-11179) - Starting with PMM 2.34.0, we support push metrics mode while updating labels.
+- [PMM-11189](https://jira.percona.com/browse/PMM-11189) - Starting With PMM 2.34.0, we have implemented vmproxy integration and frontend API changes.
+- [PMM-10705](https://jira.percona.com/browse/PMM-10705) - As of PMM 2.34.0, Operator Lifecycle Management (OLM) is used for handling the installation and updating of operators for PMM in an effective, automated, and scalable manner.
+
+## Bugs Fixed
+
+- [PMM-7350](https://jira.percona.com/browse/PMM-7350) - [DbaaS]: Fixed an issue where DB Cluster did not restart all PXC servers and proxy SQL servers on restarting.
```
Comment (denisok) [inline docs/release-notes/2.34.0.md:69]:
```suggestion
- [PMM-7350](https://jira.percona.com/browse/PMM-7350) - [DBaaS]: Fixed an issue where DB Cluster did not restart all PXC servers and proxy SQL servers on restarting.
```

File: docs/release-notes/2.34.0.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+- [PMM-7958](https://jira.percona.com/browse/PMM-7958) - During Postgres monitoring, databases were not getting deleted. This issue has been resolved now.
+- [PMM-9505](https://jira.percona.com/browse/PMM-9505) - [DBaaS]: The options to update, restart, and view logs were enabled for paused clusters. This issue has now been resolved by disabling these options.
+- [PMM-9977](https://jira.percona.com/browse/PMM-9505) - [Backup]: When deleting a backup artifact that was expanded from **Backup > All Backups**, PMM would automatically expand the next row in the table. This issue is now fixed and PMM refreshes the list of backups without automatically expanding any rows.
+- [PMM-10853](https://jira.percona.com/browse/PMM-10853) - [Alerting] : Duplicate Buttons for Saving rule based on Percona Templates
+- [PMM-10871](https://jira.percona.com/browse/PMM-10871) - [Error 404]: When clicking on Alerting under Settings/Alertmanager Integration
```
Comment (denisok) [inline docs/release-notes/2.34.0.md:74]:
```suggestion
- [PMM-10871](https://jira.percona.com/browse/PMM-10871) - [Alerting]: Error 404, when clicking on Alerting under Settings/Alertmanager Integration
```

File: docs/release-notes/2.34.0.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+- [PMM-10942](https://jira.percona.com/browse/PMM-10942) - [PITR] - is possible to pass Schedule for every hour without specific minute
```
Comment (denisok) [inline docs/release-notes/2.34.0.md:75]:
```suggestion
- [PMM-10942](https://jira.percona.com/browse/PMM-10942) - [Backup] - PITR: is possible to pass Schedule for every hour without specific minute
```

File: docs/release-notes/2.34.0.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+- [PMM-11049](https://jira.percona.com/browse/PMM-11049) - Fixed a typo that was present in enhanced metrics scraper for rds_exporter.
+- [PMM-11137](https://jira.percona.com/browse/PMM-11137) - An issue with the MongoDB exporter not showing the version for MongoDB instance has been resolved.
+- [PMM-11210](https://jira.percona.com/browse/PMM-11210) - Whenever a cluster was unavailable or deleted, the K8s cluster list was empty. We have resolved this issue.
+- [PMM-11221](https://jira.percona.com/browse/PMM-11221) - [DBaaS]: Fixed an issue where the page did not load when DBaaS was disabled.
+- [PMM-11239](https://jira.percona.com/browse/PMM-11239) - Fixed an issue where the `pmm-server `package was not removed during upgrade.
+- [PMM-11372](https://jira.percona.com/browse/PMM-11239) - Despite removing pmm-server packages before upgrading PMM server, the packages were still present after upgrading. The issue has been resolved now.
+- [PMM-11333](https://jira.percona.com/browse/PMM-11333) - Fixed an issue where the API key was missing on K8s cluster registration after moving to Operator Lifecycle Management (OLM).
```
Comment (denisok) [inline docs/release-notes/2.34.0.md:82]:
```suggestion
- [PMM-11333](https://jira.percona.com/browse/PMM-11333) - [DBaaS]: Fixed an issue where the API key was missing on K8s cluster registration after moving to Operator Lifecycle Management (OLM).
```

Comment (denisok) [reply docs/release-notes/2.34.0.md:82]: I think this is intenral defect, @gen1us2k should we just remove it from RNs?

**Review by denisok — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.34.0.md (hunk 10 above)
Comment (rasika-chivate) [reply docs/release-notes/2.34.0.md:30]: The documentation is not yet ready. Will add the link when the doc is ready.

File: docs/release-notes/2.34.0.md (hunk 5 above)
Comment (rasika-chivate) [reply docs/release-notes/2.34.0.md]: ok

**Review by gen1us2k — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Review by PavelKhripkov — APPROVED** [review]

**Pull request merged**